### PR TITLE
"any" value added to target option

### LINKED ISSUE #95: Does not work for git submodules
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the regression has not already been reported

### Last working version

2.1.1

### Stopped working in version

2.2.0

### Node.js version

N/A

### Operating system

Linux

### Operating system version (i.e. 20.04, 11.3, 10)

20.04

### 💥 Regression Report

Updating git submodules stopped working in 2.2.0 due to the newly added `target` parameter. Submodules are updated by new commits, and so they fail the SemVer check (compared by hashes):

Workflow error

### Steps to Reproduce

- Add a submodule in your repo
- Create a dependabot workflow to update submodules:
```yml
version: 2
updates:
  - package-ecosystem: "gitsubmodule"
    directory: "/"
    schedule:
      interval: "daily"
```
- Create a simple GitHub action for `fastify/github-action-merge-dependabot`

### Expected Behavior

It should just ignore git submodules and automatically merge them regardless. A `skip` option for `target` would be a solution for that.

### PR DESCRIPTION
**Pull request by sameer-coder**
The target value can be one of the `major, premajor, minor, preminor, patch, prepatch or prerelease`. This PR adds another value `any` which can be used to run the action without any semver version checks.

Documentation has been updated

Closes #95

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -45,10 +45,12 @@ async function run() {
       return logWarning('Not a dependabot PR, skipping.')
     }
 
-    const isTargetMatchToPR = checkTargetMatchToPR(pr.title, TARGET)
+    if (TARGET !== 'any') {
```
Comment (simoneb) [inline src/index.js:48]: let's use the constants

**Review by simoneb — COMMENTED** [review]
Shall we make any the new default?

**Comment by sameer-coder** [comment]
yeah I think `any` as the new default makes more sense

**Comment by sameer-coder** [comment]
@simoneb is this PR good to merge? I had updated the test cases.

**Review by simoneb — CHANGES_REQUESTED** [review]
They way you updated the tests doesn't look right, the structure of the nested tests specifically. Please have a look

**Comment by sameer-coder** [comment]
> They way you updated the tests doesn't look right, the structure of the nested tests specifically. Please have a look

The nested tests were already there. I have eliminated the nesting and improved the structure now.

**Review by simoneb — APPROVED** [review]
The nesting made sense, it just didn't make much sense after you had changed it.

It made sense because previously we returned major when:

- major was provided
- input was not recognized

With your change "any" would have got the same behavior as major had previously but you didn't update the structure of the tests accordingly.

Without nesting the problem does not exist in the first place so we're good

**Pull request merged**